src/settings_type.h

```diff
@@ -549,6 +549,8 @@ struct EconomySettings {
 	uint16_t town_noise_population[4];         ///< population to base decision on noise evaluation (@see town_council_tolerance)
 	bool   allow_town_level_crossings;       ///< towns are allowed to build level crossings
 	bool   infrastructure_maintenance;       ///< enable monthly maintenance fee for owner infrastructure
+	uint16_t town_cargo_scale_percentage;     ///< scale cargo production of towns by this percentage.
+	uint16_t industry_cargo_scale_percentage; ///< scale cargo production of industries by this percentage.
```

Comment:
Is `_percentage` necessary? Very long variable names... For interface scaling, which is a percentage, I went with just `gui_scale`.